### DISCUSSION TITLE
Watchtower Stop node-exporter an Prometheus and does not restart it

### DISCUSSION THREAD

**Discussion by ragman1976** [containrrr/watchtower#829] [Q&A] [answered]
Hello,

as I mentioned above watchtower stop nodes-exporter and Prometheus container and does not restart it.
Running watchtower on an RaspiPi4 in Docker/Poertainer enviroment.

Whats going wrong?

Greets

**Comment by piksel** [comment, marked as answer]
Have you checked the logs? You can increase the verbosity with the `--debug` command flag, or `WATCHTOWER_DEBUG` environment variable. It does sound like it crashes though, in which case there should be a panic trace regardless of error level.
